src/configs/esbuild.es5.config.js

```diff
@@ -51,6 +51,7 @@ module.exports = (folder, globalName) => {
       alias([
         { find: '@', filter: /@\//, replace: path.resolve(process.cwd(), 'src/') },
         { find: '~', filter: /~\//, replace: path.resolve(process.cwd(), 'node_modules/') },
+        { find: '@lightningjs/core', filter: /^@lightningjs\/core$/, replace: path.join(__dirname, '../alias/lightningjs-core.js'),},
```

Comment:
Thanks for finding this one and bringing it up to par with the rollup version!

Reply:
You are welcome.